apps/explorer/src/pages/transaction-result/programmable-transaction-view/InputsCard.tsx

```diff
@@ -41,7 +42,9 @@ export function InputsCard({ inputs }: InputsCardProps) {
 					) {
 						renderValue = <AddressLink address={stringValue} />;
 					} else if (REGEX_NUMBER.test(stringValue)) {
-						renderValue = Number(value).toLocaleString();
+						const bigNumber = new BigNumber(stringValue);
```

Comment:
qq: why not use `BigInt`?

Reply:
No strong pref, I switch to BigInt to not relying on BigNumberJs since we just displaying the value directly